backend/src/url_shortener/core/settings/environment/dev.py

```
@@ -81,7 +86,7 @@ def _extract_storage_variables(self):
 				f"Unsupported CLOUD_PROVIDER: {self.CLOUD_PROVIDER}. Use 'aws' or 'azure'"
 			)
 
-	# def _extract_firebase_variables(self):
-	#       self.USING_FIREBASE_EMULATOR = os.getenv("USING_FIREBASE_EMULATOR")
-	#       self.FB_AUTH_EMULATOR_HOST= os.getenv("FB_AUTH_EMULATOR_HOST")
-	#       self.FB_PROJECT_ID = os.getenv("FB_PROJECT_ID")
+	def _extract_firebase_variables(self):
```

Ruff (0.14.6)

Missing return type annotation for private function `_extract_firebase_variables`

Add return type annotation: `None`

(ANN202)